README.md

```diff
@@ -69,7 +69,25 @@ How to [secure your setup](/doc/security/ssl.md).
 
 ## Contributing
 
-Development guides are coming soon.
+### Getting started
```

Comment:
We already have a `Getting Started` guide, so this could be confusing. Where do you think this should be placed @nol166 ?